### PR TITLE
Implemented Carousel #84

### PR DESCRIPTION
**Pull request by Strange-Philip**

### CODE REVIEW

**Comment by Utkarsh4517** [comment]
Hey @Strange-Philip can you please provide a screenshot or a gif of the widget you made?

**Comment by Strange-Philip** [comment]
carousell
here 😊

File: lib/src/carousel/carousel.dart [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+  /// Boolean to enable/disable indicator ; default: true
+  final bool? showIndicator;
+
+  /// Boolean to enable/disable indicator ; default: true
+  final bool? showButtons;
+
+  /// Carousel Button Types
+  final CarouselButtonType? buttonType;
+  const MUICarousel(
```
Comment (Utkarsh4517) [inline lib/src/carousel/carousel.dart:41]: Place the constructor above the variables declaration, this change was suggested by @omjogani 
Also add an optional `maxWidth` param, default value should be 430. refer to our widget creation guide for more information regarding this. 
https://utkarshs-organization-1.gitbook.io/simple-guide-to-create-modularui-widgets/

else everything LGTM 👍🏼

**Review by Utkarsh4517 — CHANGES_REQUESTED** [review]
Hey just add an `maxWidth`  param and place the constructor above the variables declaration.

**Comment by Strange-Philip** [comment]
Done ... just made a commit

**Comment by Utkarsh4517** [comment]
Thank you for your valuable contribution @Strange-Philip 🫡

**Comment by Strange-Philip** [comment]
Always happy to help 😊... @Utkarsh4517

**Pull request merged**